assume/units/powerplant.py

```diff
@@ -418,24 +420,20 @@ def calc_marginal_cost_with_partial_eff(
         return marginal_cost
 
     def calculate_min_max_power(self, start: pd.Timestamp, end: pd.Timestamp):
-        base_load = self.outputs["energy"][start:end]
+        end_excl = end - self.index.freq
+        base_load = self.outputs["energy"][start:end_excl]
         # check if min_power is a series or a float
-        min_power = (
-            self.min_power[start:end].max()
-            if type(self.min_power) is pd.Series
-            else self.min_power
-        )
-        min_delta = min_power - base_load.min()
+        min_power = self.min_power - base_load.min()
```

Comment:
if my availability is 0 i can not even bid min_power right?

Reply:
true. added the check to the operation window function since if it is zero nothing can be done basically